DESCRIPTION

```diff
@@ -30,6 +30,8 @@ Suggests:
     dbplyr,
     dplyr,
     evaluate,
+    flextable,
```

Comment:
Do we **really** need this? I would like to avoid extra dependencies, and this package comes with a lot of recursive deps.

Reply:
It was the only one that gave me control over cell widths in both the HTML and PDF version.
The previous approach via a simple `kable()` looked fine in HTML but not in PDF.
The dep chain of the book is so long already (because we are pulling the whole mlr3verse), I honestly do not think that it makes a difference in here.

Reply:
{flextable} works really great for aligning our tables in the PDF. Everything else was a mess, especially when it comes to resizing individual columns.